core/subsurfacestartup.c

```diff
@@ -90,8 +90,8 @@ struct preferences default_prefs = {
 	},
 	.planner_deco_mode = BUEHLMANN,
 	.vpmb_conservatism = 3,
-	.distance_threshold = 1000,
-	.time_threshold = 600,
+	.distance_threshold = 50,
```

Comment:
That may be a bit aggressive (and hard on the battery). A dive boat on a buoy could create +/- 50m variability while basically "in place".
I think there is no "right" value... I would have guessed that 100m might be a better compromise. 
It seems silly to ask you to re-factor this, so I'll just make the change when I pull this (I hope you don't mind)